README.md

```diff
@@ -1,21 +1,83 @@
-[![GitHub release](https://img.shields.io/github/release/cybozu-go/neco-template.svg?maxAge=60)][releases]
-[![CircleCI](https://circleci.com/gh/cybozu-go/neco-template.svg?style=svg)](https://circleci.com/gh/cybozu-go/neco-template)
-[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/neco-template?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/neco-template?tab=overview)
-[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/neco-template)](https://goreportcard.com/report/github.com/cybozu-go/neco-template)
+[![GitHub release](https://img.shields.io/github/release/cybozu-go/pod-security-admission.svg?maxAge=60)][releases]
+[![CircleCI](https://circleci.com/gh/cybozu-go/pod-security-admission.svg?style=svg)](https://circleci.com/gh/cybozu-go/pod-security-admission)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/pod-security-admission?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/pod-security-admission?tab=overview)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/pod-security-admission)](https://goreportcard.com/report/github.com/cybozu-go/pod-security-admission)
 
-Template repository for Neco
-============================
+Pod Security Admission
+======================
 
 **Project Status**: Initial development
 
+`pod-security-admission` is a Kubernetes [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) to ensure [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
```

Comment:
"is composed of validation and mutating admission webhooks" looks better than "**a** ... webhook**s**"  
 for me.